main.js

```diff
@@ -110,7 +110,11 @@ async function real_main(options={}) {
 
     await render.doRender(config, results);
     if (!config.keep_open) {
-        const anyErrors = results.tests.some(s => s.status === 'error' && !s.expectedToFail);
+        const anyErrors = results.tests.some(
+            s =>
+                (s.status === 'error' && !s.expectedToFail) ||
```

Comment:
We discussed this before. Since `expectedToFail` can be applied to flaky tests, we definitely wanted to error on the side of green and just output something for these tests.

Reply:
Alright, I'll close this.